src/java/org/apache/cassandra/io/sstable/SSTableMultiWriter.java

```diff
@@ -40,7 +40,13 @@ public interface SSTableMultiWriter extends Transactional
     Collection<SSTableReader> finish(boolean openResult);
     Collection<SSTableReader> finished();
 
-    SSTableMultiWriter setOpenResult(boolean openResult);
+    /**
```

Comment:
So in the NSS case this ensures the data is on S3 before opening?

Reply:
Correct, that's the idea.